src/Persistence.LinnApps/Repositories/InterCompanyInvoiceRepository.cs

```diff
@@ -30,5 +30,20 @@ public IQueryable<InterCompanyInvoice> FindAll()
         {
             throw new NotImplementedException();
         }
+
+        public InterCompanyInvoice FindById(InterCompanyInvoiceKey key)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void Add(InterCompanyInvoice entity)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void Remove(InterCompanyInvoice entity)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
Why change to an IRepository if none of the new methods are implemented? Or is this for the future?

Reply:
Can I still pull them in with exportReturnDetails as an IQueryRepo? If so then you make a v good point and I'll change back, I did it while I was trying everything under the sun to try and get that relationship working and forgot to check if it was actually needed

Reply:
Actually will check with Andrew whether he's going to be doing anything with intercompany invoices in stores that will use it soon, otherwise will get revert (unless it breaks but think you're right should be fine)